pkg/sql/exec_util.go

```diff
@@ -1057,6 +1057,17 @@ func golangFillQueryArguments(args ...interface{}) (tree.Datums, error) {
 				switch {
 				case val.IsNil():
 					d = tree.DNull
+				case val.Type().Elem().Kind() == reflect.String:
+					a := tree.NewDArray(types.String)
+					// TODO(angelaw): Ranging through val seems to require pointer arithmetic so did it this way
+					for s := range args {
+						// It is a bit hacky though...
+						if err := a.Append(tree.NewDString(reflect.ValueOf(s).String())); err != nil {
```

Comment:
Print debugging — this line doesn't end up appending what I want as `s` just becomes `ARRAY['<int Value>']`

Reply:
ah yeah, since `for s := range args` will just loop over the indexes of `args`